i18n/locales/fr.json

```diff
@@ -0,0 +1,3 @@
+{
+  "AssetsPage.title": "Oui Oui, I totally know French"
+}
```

Comment:
But the translation strings are just `"id": "translated string"` pairs in JSON. Note that this is a totally fake translation, I don't actually know 
français. But I figure the transifex step shouldn't be a hugely difficult problem, yeah?